### PR TITLE
Development & software operations

### PR DESCRIPTION
**Pull request by yssavchuk**

### CODE REVIEW

**Comment by yssavchuk** [comment]
- [ ] Dockerfile
- [ ] Custom database env and database.yml
- [ ] Generate tls secret from host
- [ ] Remove code duplication use _helpers.tpl

**Comment by yivo** [comment]
@yssavchuk Please, don't include Node.js in Docker images. Also use latest Ruby – 2.5.

**Comment by yivo** [comment]
@yssavchuk Please, allow to customize RAILS_ENV in Docker image. Always set RAILS_ENV.

**Comment by yivo** [comment]
@yssavchuk You can check peatio repo on how to solve these issues.

**Comment by yssavchuk** [comment]
@yivo okay thanks

File: config/database.yml [hunk 1]
```diff
@@ -21,34 +10,12 @@ development:
...
-#
-#   production:
-#     url: <%= ENV['DATABASE_URL'] %>
-#
 production:
   <<: *default
-  database: barong_production
-  username: barong
-  password: <%= ENV['BARONG_DATABASE_PASSWORD'] %>
+  database: <%= ENV['DATABASE_NAME'] %>
```
Comment (yivo) [inline config/database.yml]: @yssavchuk Please, use `ENV.fetch` if you require value.

File: config/charts/barong/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  value: {{ .Values.db.host }}
+- name: DATABASE_USER
+  value: {{ default "root" .Values.db.user }}
+- name: SECRET_KEY_BASE
+  valueFrom:
+    secretKeyRef:
+      name: {{ template "fullname" . }}
+      key: cookiesSecretKey
+{{- if .Values.db.password }}
+- name: DATABASE_PASS
```
Comment (yivo) [inline config/charts/barong/templates/_helpers.tpl]: @yssavchuk It is much better to use DATABASE_PASS!

**Comment by yivo** [comment]
@yssavchuk Please, rebase.

**Pull request merged**